korangar/src/loaders/model/mod.rs

```diff
@@ -390,27 +455,36 @@ impl ModelLoader {
 
         let mut bounding_box = AABB::uninitialized();
         let root_node = Self::process_node_mesh(
+            version,
             root_node,
             &model_data.nodes,
             &mut processed_node_indices,
             vertex_offset,
             &mut native_model_vertices,
             &texture_mapping,
+            &hashmap_texture,
             &Matrix4::identity(),
             &mut bounding_box,
             reverse_order,
             model_data.shade_type == 2,
             model_data.animation_length,
         );
 
+        drop(hashmap_texture);
+
         let mut root_nodes = vec![root_node];
-        let is_static = root_nodes.iter().all(Self::is_static);
+        let is_static = match version.equals_or_above(2, 2) {
+            // TODO: Need to add the dynamic RSM2.
```

Comment:
Is this TODO still open?

Reply:
It is still open, I am trying to make the dynamic work.

Reply:
Sure, but I am not finished with dynamic models yet